src/main/main.py

```diff
@@ -1,129 +1,47 @@
-from src.main import consts, activity_tracker_handler as ath, code_tracker_handler as dh
-import pandas as pd
+import datetime
 import logging
-import csv
-import sys
 import os
+import sys
+
+import pandas as pd
+
+sys.path.append('.')
+from src.main.splitting.splitting import run_tests
+from src.main.util import consts
+from src.main.util.consts import PATH_CMD_ARG
+from src.main.util.file_util import add_slash
 
 pd.set_option('display.max_rows', 250)
 pd.set_option('display.max_columns', 100)
 
+log = logging.getLogger(consts.LOGGER_NAME)
+
 
 def __get_data_path():
     args = sys.argv
-    if len(args) < 3 or consts.PATH_CMD_ARG not in args:
-        raise NameError('List of arguments must have required 1 element: path!')
-    try:
-        path = args[args.index(consts.PATH_CMD_ARG) + 1]
-    except ValueError:
-        raise NameError('List of arguments does not have the argument -path')
-    except IndexError:
-        raise NameError('List of arguments does not have a value after the argument -path')
+    path = args[args.index(PATH_CMD_ARG) + 1]
+    if not os.path.isdir(path):
+        log.error("It's not a folder, passed path is " + path)
+        sys.exit(1)
+    path = add_slash(path)
     return path
```

Comment:
Maybe is it better to return add_slash(path) without an additional variable path?